src/mds/topology/topology_service_manager.cpp

```diff
@@ -55,9 +55,21 @@ using ::curve::mds::copyset::Copyset;
 using ::curve::mds::copyset::CopysetConstrait;
 using ::curve::common::CopysetInfo;
 
+
 void TopologyServiceManager::RegistChunkServer(
     const ChunkServerRegistRequest *request,
     ChunkServerRegistResponse *response) {
+    if (request->has_blocksize()) {
```

Comment:
Consider that chunkserver gets blocksize from mds instead of configuring blocksize individually and then verifying